src/main/resources/assets/betterquesting/lang/en_US.lang

```diff
@@ -87,10 +88,16 @@ betterquesting.gui.closing_warning=Are you sure you want to close this editor?
 betterquesting.gui.closing_confirm=Changes may be lost!
 betterquesting.gui.claim_all_warning=Are you sure you want to claim all rewards?
 betterquesting.gui.claim_all_confirm=There may be a lot of items that will drop to the ground if your inventory is full!
+betterquesting.gui.claim_all_quest_count=Quests to claim: %d
+betterquesting.gui.claim_all_choice_quest_count=Available choice rewards: %d
 betterquesting.gui.key=Key Name
 betterquesting.gui.no_key=No Key
 betterquesting.gui.duplicate_key=Duplicate Key
 betterquesting.gui.yes_always=Yes, always
+betterquesting.gui.force_choice_yes=Choices: §aYes
```

Comment:
This could potentially be misleading if for some reason the player ignores the tooltip and assumes that "Choices: Yes" means they will get to make choices, whereas in fact we are making the choices for them. If we need to make this specific button larger in order to have something a bit less ambiguous, it might be worth it. Not a blocker at all though.

Reply:
Not sure, I had some trouble figuring out the best way to display this in a small amount of text that doesn't overflow outside of the button size. I think the button tooltip makes it clear though